spec/spec_helper.rb

```diff
@@ -1 +1,6 @@
 require 'nexpose'
+
+if ENV['CODECLIMATE_REPO_TOKEN']
+  require 'codeclimate-test-reporter'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.